sources/osg/Matrix.js

```diff
@@ -604,6 +640,24 @@ var Matrix = {
         };
     } )(),
 
+    preMultRotate: function ( matrix, quat ) {
+        if ( Quat.zeroRotation( quat ) )
```

Comment:
not sure that we want to test each time the zero rotationI would prefer raw code instead of checking each time here